server/vulnerabilities/oval/parsed/host_package.go

```diff
@@ -0,0 +1,6 @@
+package oval_parsed
+
+type HostPackage struct {
```

Comment:
Please document, assuming this will be the data fetched from Ubuntu hosts.

(Maybe rename to `UbuntuHostPackage`?, no idea if we'll need more/other information for `CentOS` packages.)

Reply:
That type will be removed in the next PR - that was added just as a quick and easy way to test the evaluation part of this.